frontend/src/app/components/enterprise/enterprise-base.component.sass

```diff
@@ -1,7 +1,8 @@
-.button
-  float: left
+.op-enterprise-base
+  .button
+    float: left
 
-.confirmation-hint
-  font-size: 0.9em
-  font-style: italic
-  margin-bottom: 2rem
+  .confirmation-hint
```

Comment:
if confirmation hint is only used in this file, then make it an element of the block (e.g. `&--hint` or `&--confirmation-hint`)

Reply:
If it is used in more places than this file, perhaps we should add a TODO here to make sure we also refactor the confirmation hint  in the future

Reply:
Hey @b12f , to make `confirmation-hint` as an element of the block, should i use -- or __ ? `&--confirmation-hint`  or `&__confirmation-hint`?

Reply:
`--` is for blocks , a single `_` is for modifiers